src/plugins/vis_type_timelion/public/components/area_series/index.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { AreaSeries, ScaleType, CurveType } from '@elastic/charts';
+
+interface AreaSeriesComponentProps {
+  data: any;
+  index: number;
+}
+
+export function AreaSeriesComponent({ data, index }: AreaSeriesComponentProps) {
+  const lines = data.lines || {};
+  const points = data.points || {};
+  const styles = {
+    areaSeriesStyle: {
+      line: {
+        stroke: data.color,
+        strokeWidth: Number(lines.lineWidth) || 3,
+        visible: lines.show === undefined ? true : lines.show,
+      },
+      area: {
+        fill: data.color,
+        opacity: !lines.fill || lines.fill <= 0 ? 0 : lines.fill,
+        visible: lines.show === undefined ? true : lines.show,
+      },
+      point: {
+        fill: points.fillColor,
+        opacity: !points.fill || points.fill < 0 ? 1 : points.fill,
```

Comment:
can we do something like `points.fill ?? 1` here?

Reply:
Unfortunately, no. We cannot set negative values.

Reply:
@VladLasitsa if the fill is `0` that is a valid number will be automatically changed to 1 with this statement. I think we should let the user set `0` as valid opacity

Reply:
now it simplified